packages/plugin-core/src/server.ts

```diff
@@ -0,0 +1,14 @@
+import { ServerUtils } from "@dendronhq/api-server";
+import { stringifyError } from "@dendronhq/common-all";
+
+(async () => {
```

Comment:
nit: since this function/js file isn't explicitly referenced, can we add a comment around how this is intended to be used and/or that it's invoked from execa in _extension.ts